src/lib/ColonyManager/ColonyManager.ts

```diff
@@ -121,23 +136,23 @@ export default class ColonyManager {
         return this.networkClient;
       }
       case ClientType.ColonyClient: {
+        if (!identifier)
```

Comment:
shouldn't we just make it required if this is required for all clients? 🤔

Reply:
I'm not sure. This came from one of the commits I cherry-picked from #2260, commit 2d68aead5394038d4712209bbbb0061d96346fb6

I guess I'll have to look into it :+1:

Reply:
The `NetworkClient` does not require (and cannot have) an identifier, yet also can be returned from this function. This is here to make typescript happy.